flybywiresim/a32nx
# feat: update de-DE.locPak

Kurzer Hinweis zu den entfernten Anführungszeichen:

Anders, als etwa der Duden die deutsche Rechtschreibung interpretiert, können Anführungszeichen nicht innerhalb von einem Wort stehen. Hier wären sie ohnehin weniger angebracht, weil Anführungszeichen primär zum Zwecke des Zitats verwendet werden; zum bloßen Hervorheben würde man z. B. die Kursivschreibung verwenden, nicht aber Anführungszeichen.

## Summary of Changes
Spelling: Remove inappropriate use of quotation marks in the German aircraft description text. Add a hyphen (so-called _Durchkopplung_ or _Bindestrichschreibung_ in this context) for clarity in the loading screen text.

## Testing instructions

- Change simulator language to German.
- Observe the change in the aircraft description text. Observe the removed quotation mark around the word "Sharklet". Make sure there are no formatting issues. Note: I don't actually know where this shows up to be honest.
- Try finding the changed loading screen tip while loading up a flight. Observe the added hyphen. Make sure there are no formatting issues.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
